examples/example.cmake

```diff
@@ -1,5 +1,6 @@
 #[[
```

Comment:
I suspect there's a way to have Sphinx include a file verbatim in the documentation so you can keep this all synched (and unit tested) without having to have it in multiple places.

Reply:
Included the example.cmake in the quickstart instead of inline code. Also generated an example Sphinx project in the examples directory using the RST file generated from example.cmake.